VirtoCommerce.InventoryModule.Web/Controllers/Api/InventoryModuleController.cs

```diff
@@ -52,10 +47,10 @@ public IHttpActionResult SearchInventories([FromBody] InventorySearchCriteria se
         /// </summary>
         [HttpPost]
         [Route("fulfillmentcenterinventories/search")]
-        [ResponseType(typeof(GenericSearchResult<FulfillmentCenterInventoryInfo>))]
+        [ResponseType(typeof(GenericSearchResult<InventoryInfo>))]
```

Comment:
We do not need this API endpoint now, please remove it and use `inventories/search` with the AllFFC response group.
Please also remove the corresponding method from the Angular resource.

Reply:
done